crates/ursa-network/Cargo.toml

```diff
@@ -10,6 +10,8 @@ description = "Ursa's libp2p implementation"
 [dependencies]
 anyhow = "1.0.56"
 async-fs = "1.6.0"
+async-std = { version = "1.11.0", features = ["attributes"] }
```

Comment:
Do we need a async-std for any feature?

Reply:
Sorry I forgot to remove this we moved to Tokio.